ckan/logic/action/get.py

```diff
@@ -1153,6 +1154,33 @@ def resource_view_list(context, data_dict):
     return model_dictize.resource_view_list_dictize(resource_views, context)
 
 
+def resource_file_metadata_show(context, data_dict):
+    '''Get file metadata from a resource if it was uploaded.
```

Comment:
Should there be a special case for a resource that exists but wasn't uploaded?

Reply:
if its url based then its not_found since this should be called per package/resource show